### PR TITLE
Create user model

### PR DESCRIPTION
**Pull request by EduardoContreras**
**Summary**
Was added create user model into app and generated the same model to data base with sequelize orm.

**Trello Card**
https://trello.com/c/AOx5qsaA/27-modelo-de-usuario

### CODE REVIEW

File: training/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DB_HOST=localhost
+DB_PORT=5432
+DB_NAME=mycooldb
+DB_USERNAME=eduardocontreras
+DB_PASSWORD=
```
Comment (santiago-alvarez-ruiz) [inline training/.env]: You don't need commit this file.

File: training/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      "strict": 0,                                                  // http://eslint.org/docs/rules/strict
+      "valid-jsdoc": 0,                                             // http://eslint.org/docs/rules/valid-jsdoc
+      "vars-on-top": 0,                                             // http://eslint.org/docs/rules/vars-on-top
+      "wrap-regex": 0,                                              // http://eslint.org/docs/rules/wrap-regex
+  
+  
+  
+      "prettier/prettier": ["error", { "printWidth": 110, "singleQuote": true }]         // https://github.com/prettier/prettier
+    }
+  }
```
Comment (santiago-alvarez-ruiz) [inline training/.eslintrc.json]: Missing newline.

File: training/.sequelizerc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('config', 'db.js'),
+  'migrations-path': path.resolve('migrations', 'migrations'),
+  'models-path': path.resolve('app', 'models')
+};
```
Comment (santiago-alvarez-ruiz) [inline training/.sequelizerc]: Same.

**Review by santiago-alvarez-ruiz — COMMENTED** [review]
After fix those little comments considered this PR approved.

File: training/.env (hunk 1 above)
Comment (EduardoContreras) [reply training/.env]: done

File: training/.eslintrc.json (hunk 2 above)
Comment (EduardoContreras) [reply training/.eslintrc.json]: done

File: training/.sequelizerc (hunk 3 above)
Comment (EduardoContreras) [reply training/.sequelizerc]: done

**Comment by Atixx** [comment]
change the base branch to the one you made your first commit from, so only the relevant changes are shown

**Comment by gonziesc** [comment]
Change the base in order to see only the branch changes

**Comment by EduardoContreras** [comment]
Was changed for https://github.com/wolox-training/edc-express-js/pull/4

**Pull request closed without merging**